app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -207,6 +207,8 @@ function insertArticles(articles) {
   var list = document.getElementById('substories');
   var newArticlesHTML = '';
   var el = document.getElementById('home-articles-object');
+  var currentUser = userData();
```

Comment:
@aitchiss I know you mentioned depending on `getUserDataAndCsrfToken` for the `currentUser` data. However, it seemed to me that `userData()` is available in this file, so i used it instead. Please advise. 🙏🏾

Reply:
Ah yes, good point, we're in initializer code here so it's not so easy to use the `getUserDataAndCsrfToken` helper. Sounds good then - let's rely on `userData`. It should be safe here because it's not being called immediately as the page loads (so we can be pretty much sure the userData already exists)